### ISSUE TITLE
Questions about Adaptive S-Norm for ASV

### ISSUE THREAD

**Issue by CaptainPrice2023** [sinhat98/adapter-wavlm#8]
Thanks for sharing the code. In your paper, it mentioned that the Adaptive S-Norm was applied. But implementation in the [utils.py](https://github.com/sinhat98/adapter-wavlm/blob/main/ASV/utils.py) is different from AS-Norm. The code didn't choose some top n cohort but use all (10000 in the code) from the training set to compute the mean and std. It looks like S-Norm, not AS-Norm. I was wondering did you have a chance to compare S-Norm vs AS-Norm in the experiment? Will you add AS-Norm into the code as well?

**Comment by sinhat98** [comment]
Thank you for your feedback regarding our paper and repository. We acknowledge the inaccuracies in the paper regarding the measurement and comparison of adaptive s-norms. It was not part of our study, and we do not intend to include adaptive s-norms in the associated repository.

Your attention to detail is appreciated, and we apologize for any confusion caused by these inaccuracies.

**Comment by CaptainPrice2023** [comment]
Thanks for your quick reply!

Here is another problem I met recently. For ASV task, if I replace the Cross entropy loss with the commonly used AMSoftmax, the training process will get some problems, training loss cannot converge and training ACC keeps very low (lower than 1%). This is very weird. I was wondering if you have tried using AMSoftmax and got a similar observation. Thanks.
